mass_mailing_batch/models/mass_mailing.py

```diff
@@ -0,0 +1,29 @@
+#  Copyright 2023 Simone Rubino - TAKOBI
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class MassMailing (models.Model):
+    _inherit = 'mail.mass_mailing'
+
+    def get_remaining_recipients(self):
+        recipients_res_ids = super(MassMailing, self).get_remaining_recipients()
+        batch_size = self.env['ir.config_parameter'] \
+            .sudo() \
+            .get_param(
+                'mass_mailing_batch.size',
```

Comment:
Thanks for the review!
I prefer to keep the module name in the parameter key, in order to avoid clashing with other parameter names.
If you really don't like the key right now, maybe I could change it to `mass_mailing_batch.batch_size`, what do you think?

Reply:
OK `mass_mailing_batch.size`